### PR TITLE
feat: sepolia deployment

### PR DESCRIPTION
**Pull request by thomas192**
- [x] issue #130
- [x] follows contribution [guide](https://github.com/keep-starknet-strange/blobstream-starknet/blob/main/CONTRIBUTING.md)
- [ ] code change includes tests
- [ ] breaking change

Created a bash deployment script for blobstreamx contract, accompanied by documentation.

### CODE REVIEW

File: scripts/deploy-blobstreamx.sh [hunk 1]
```diff
@@ -0,0 +1,188 @@
+#!/bin/bash
+source ../.env
```
Comment (drspacemn) [inline scripts/deploy-blobstreamx.sh:2]: As `starkli` uses these same environment variables the dev may have them in the env already. Lets parse this `.env` only if the environment variables aren't already set.

Comment (drspacemn) [reply scripts/deploy-blobstreamx.sh:2]: This will also make the script less path dependent

**Comment by drspacemn** [comment]
Can you provide the values for `--height` and `---header` that you used to test

File: scripts/deploy-blobstreamx.sh (hunk 1 above)
Comment (thomas192) [reply scripts/deploy-blobstreamx.sh]: Made the change

**Comment by thomas192** [comment]
> Can you provide the values for `--height` and `---header` that you used to test

I used random values  for `--height` and `--header` to test the deployment script. I figured it didn't matter at this point. May be missing something, could you explain ?

**Comment by drspacemn** [comment]
> > Can you provide the values for `--height` and `---header` that you used to test
> 
> I used random values for `--height` and `--header` to test the deployment script. I figured it didn't matter at this point. May be missing something, could you explain ?

For the following deploy 

`bash scripts/deploy-blobstreamx.sh --owner 0x0583e9857d5547330859698f711a7bd354d88970cb8987ea4c6b084ce3567319 --height 1535338 --header 0x554EEB0B9039742D192D9AD7389C3E556AC3A20D9695A3C3A4B300A8EF6253D4`

I get the error:

`Deployed at: Error: Error: invalid character`

Appears the script is expecting a different format for one of the arguments?

**Comment by drspacemn** [comment]
UPDATE: Checking w/ `--debug` looks to be user error on my part

File: scripts/deploy-blobstreamx.sh [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+if [ -z "$STARKNET_KEYSTORE" ]; then
+  echo "Error: STARKNET_KEYSTORE is not set."
+  exit 1
+elif [ -z "$STARKNET_ACCOUNT" ]; then
+  echo "Error: STARKNET_ACCOUNT is not set."
+  exit 1
+fi
+
+# Constants
+SIERRA_FILE=../target/dev/blobstream_sn_blobstreamx.contract_class.json
```
Comment (drspacemn) [inline scripts/deploy-blobstreamx.sh:18]: I usually run the scripts from the top level dir. Can we make this less path dependent? Maybe `git rev-parse --show-toplevel`

Comment (thomas192) [reply scripts/deploy-blobstreamx.sh:18]: Made the change in #137

**Pull request merged**